infrastructure/terraform/web-server-service.tf

```diff
@@ -0,0 +1,43 @@
+module "web_server_rabbitmq_user" {
+  source         = "modules/rabbitmq-user"
+  prefix         = "${var.prefix}"
+  project_name   = "taskcluster-web-server"
+  rabbitmq_vhost = "${var.rabbitmq_vhost}"
+}
+
+resource "random_string" "web_server_access_token" {
+  length           = 65
+  override_special = "_-"
+}
```

Comment:
Note that if we want to give scopes to this clientId, that happens in `auth-service.tf`.  For the moment, with no login strategies enabled, this isn't important.